project/threads/templates/threads/partials/user-setup/step1.html

```diff
@@ -55,11 +55,9 @@
                             <div class="row">
                                 <div class="white-background-input-field input-field col s6">
                                     <input id="username" type="text" class="validate" value="{{username}}" disabled>
-                                    <label for="username">Username</label>
                                 </div>
```

Comment:
@gorkemarslan . I think rather then removing labels all together we could add the class `active` on to resolve the issue.
It would be better for consistency and accessibility.

Reply:
It would be better, you're right. With my solution, labels disappear. In your solution, users can understand which is which for username and email fields. This would be much clearer. Thanks!

Reply:
Yes. We need to keep the labels for the accessibility of the form.